shared/constants/chat2/index.js

```diff
@@ -216,7 +216,7 @@ export const makeInboxQuery = (
     convIDs: convIDKeys.map(Types.keyToConversationID),
     readOnly: false,
     status: Object.keys(RPCChatTypes.commonConversationStatus)
-      .filter(k => !['ignored', 'blocked', 'reported'].includes(k))
+      .filter(k => !['blocked', 'reported'].includes(k))
```

Comment:
I don't think we should do this, is this just so these show up in "jump to conv"?

Reply:
But then you couldn't get the conv back from the gui

Reply:
JUst go to new conversation.